src/main/java/com/arpnetworking/tsdcore/statistics/TPStatistic.java

```diff
@@ -93,13 +92,7 @@ public Quantity calculate(final List<Quantity> orderedValues) {
      */
     @Override
     public Quantity calculateAggregations(final List<AggregatedData> aggregations) {
-        final List<Quantity> allSamples = Lists.newArrayList();
-        for (final AggregatedData aggregation : aggregations) {
-            allSamples.addAll(aggregation.getSamples());
-        }
-        Collections.sort(allSamples);
-        final int index = (int) (Math.ceil((_percentile / 100.0) * (allSamples.size() - 1)));
-        return allSamples.get(index);
+        throw new UnsupportedOperationException("Unsupported operation: calculateAggregations(List<AggregatedData>)");
```

Comment:
Why not just remove all the calculateAggregations?  That is now the job of cagg in a completely separate code base.